File: frontend/src/types/market.ts
export interface MarketSentiment {
  score: number
  label: string
  description: string
}

export interface MarketIndex {
  name: string
  value: string
  change: string
  data: number[]
}

export interface HeatmapItem {
  name: string
  value: number
  change: number
}

File: backend/app/schemas/market.py
from pydantic import BaseModel


class MarketSentimentResponse(BaseModel):
    score: int
    label: str
    description: str


class MarketIndexItem(BaseModel):
    name: str
    value: str
    change: str
    data: list[float]


class HeatmapItem(BaseModel):
    name: str
    value: float
    change: float
